react/features/analytics/functions.js

```diff
@@ -16,7 +16,11 @@ const logger = require('jitsi-meet-logger').getLogger(__filename);
  * @returns {void}
  */
 export function sendAnalytics(event: Object) {
-    analytics.sendEvent(event);
+    try {
+        analytics.sendEvent(event);
+    } catch (e) {
+        logger.warn(`Error sending analytics event: ${e}`);
```

Comment:
have you checked what is logged when an error happens (the one you reported on the chat for example), will it be anything useful if printed this way (to string) ? I just wonder if it won't end up something like "Object [object]"

Reply:
Note that what I added i slogging the exception, not the event. This is what we get now:

![img_0019](https://user-images.githubusercontent.com/317464/35053848-9b4403f0-fbab-11e7-954a-eba930930895.PNG)